tests/rna_io/CMakeLists.txt

```diff
@@ -15,7 +15,9 @@ message (STATUS "Configuring tests/rna_io")
 # ----------------------------------------------------------------------------
 
 # Search SeqAn and select dependencies.
-find_package (SeqAn REQUIRED)
+if (NOT "${SEQAN_BUILD_SYSTEM}" STREQUAL "DEVELOP")
+    find_package (SeqAn REQUIRED)
+endif ()
```

Comment:
Why is linking against SeqAn needed in the first place? 
Doesn't the test build system take care of it?

Reply:
This is the standard for all our tests and apps. In theory we could remove it for the tests, because they are always built "internally", but I see no point in changing it now.
This diff just brings rna_io in line with all other tests.